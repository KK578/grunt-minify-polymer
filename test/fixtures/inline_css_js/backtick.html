<!DOCTYPE>
<html>
    <head>
        <title>Hello.</title>
        <meta type="charset" content="utf-8" />
    </head>
    <body>
        <script>
        function foobar() {
            var a = 5;
            var string = `Templating ${a} strings`;
            return string;
        }
        </script>
    </body>
</html>
