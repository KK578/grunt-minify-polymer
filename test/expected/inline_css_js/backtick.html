<!DOCTYPE><html><head><title>Hello.</title><meta type="charset" content="utf-8"></head><body><script>function foobar(){var a=5,string=`Templating ${a} strings`;return string}</script></body></html>